clusterloader2/pkg/measurement/common/exec.go

```diff
@@ -62,22 +62,40 @@ func (e *execMeasurement) Execute(config *measurement.Config) ([]measurement.Sum
 	if len(command) == 0 {
 		return nil, fmt.Errorf("command is a required argument. Got empty slice instead")
 	}
-
+	retries, err := util.GetIntOrDefault(config.Params, "retries", 1)
+	if err != nil || retries < 1 {
+		return nil, fmt.Errorf("error getting retries, retries: %v, err: %v", retries, err)
+	}
+	retryIntervalStr, err := util.GetStringOrDefault(config.Params, "retryInterval", "1s")
+	if err != nil {
+		return nil, err
+	}
+	retryInterval, err := time.ParseDuration(retryIntervalStr)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
`GetDurationOrDefault` will simplify this to a single error check.

Reply:
Done (here and for timeout)